lib/resty/chash.lua

```diff
@@ -99,6 +119,7 @@ local function _precompute(nodes)
     for id, weight in pairs(nodes) do
         newnodes[id] = weight
     end
```

Comment:
Maybe we don't need the lines above any more?

Reply:
already delete this code